src/components/grid/index.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { FlexColumn } from './index'
+
+describe('The FlexColumn component', () => {
+  const testChild = <p>Testing</p>
+  let wrapper
+
+  beforeAll(() => {
+    wrapper = shallow(<FlexColumn animation={false}> {testChild}</FlexColumn>)
+  })
+
+  it('Snapshot', () => {
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  it('render', () => {
+    expect(
+      wrapper
+        .find('p')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 spaces but found 8.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4658697)